docs/pages/database-access/guides/azure-postgres-mysql.mdx

```diff
@@ -3,18 +3,6 @@ title: Database Access with Azure PostgreSQL and MySQL
 description: How to configure Teleport database access with Azure Database for PostgreSQL and MySQL.
 ---
 
-<Details
-  title="Version warning"
-  opened={true}
-  scope={["oss", "enterprise"]}
-  scopeOnly={true}
->
-Azure database auto-discovery is available starting from Teleport `10.2`. Database
-access for Azure PostgreSQL/MySQL flexible servers is available starting from
-Teleport `12.0`.
```

Comment:
I'd probably keep this one for one more major release. v12 is still relatively new.